src/pages/create-course/CreateCourse.tsx

```diff
@@ -44,7 +44,7 @@ const CreateCourse = () => {
   const onResponseError = (error: ErrorResponse) => {
     setAlert({
       severity: snackbarVariants.error,
-      message: error ? `errors.${error.code}` : ''
+      message: error ? `errors.${error.message}` : ''
```

Comment:
how it will be work for other errors for which we using translation with err.code?